XenAdmin/Wizards/NewVMWizard/Page_CpuMem.cs

```diff
@@ -106,8 +108,11 @@ public override void PageLoaded(PageLoadedDirection direction)
                     spinnerStatMax.Visible = false;
             }
 
-            comboBoxTopology.Populate(Template.VCPUs_at_startup, Template.VCPUs_max, Template.CoresPerSocket, Template.MaxCoresPerSocket);
-            PopulateVCPUs(Template.MaxVCPUsAllowed, Template.VCPUs_at_startup);
+            isVcpuHotplugSupported = Template.SupportsVcpuHotplug;

```

Comment:
Note: this test will also disallow hotplug for the "other install media" template. But I think this is unavoidable, because we can't know whether the media will contain a Windows or Linux OS.